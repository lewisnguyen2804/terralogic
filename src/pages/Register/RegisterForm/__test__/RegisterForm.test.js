import React from 'react'
import { shallow } from 'enzyme';
import { RegisterForm } from '../';

describe('test for Register Page', () => {
    const wrapper = shallow(<RegisterForm />);
    it('should render correctly', () => {
        expect(JSON.stringify(wrapper)).toMatchSnapshot();
    });
});

