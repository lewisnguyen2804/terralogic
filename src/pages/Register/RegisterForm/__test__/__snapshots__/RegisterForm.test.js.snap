// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for Register Page should render correctly 1`] = `"{}"`;
